molecule/default/molecule.yml

```diff
@@ -0,0 +1,30 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: omero-all
+    image: openmicroscopy/centos-systemd-ip
```

Comment:
Are we still maintaining this image?

For a one-node playbook you might be able to get away with using the upstream `centos/systemd` image directly as in the omero-server role:

https://github.com/ome/ansible-role-omero-server/blob/118716d7ab46f9a003fba32945a924e745a4cd5b/molecule/newdep/molecule.yml#L11-L15

Otherwise we could manage the extra dependencies in a prepare.yml playbook, see e.g. IDR/deployment:
https://github.com/IDR/deployment/blob/3f9cf70d3f97d11feecf830bb5efa6fc22c3c87e/ansible/molecule/publicidr/molecule.yml#L68-L69

Yet another alternative is to modify the generated `Dockerfile.j2`

Reply:
See 2725756. Looks like using the base image is sufficient :+1: